api/MessageEvent.json

```diff
@@ -308,6 +326,9 @@
               "chrome_android": {
                 "version_added": true
               },
+              "deno": {
+                "version_added": null
```

Comment:
Since it's pretty rare, I'm OK with this being `null` for now, but we are aiming for 100% "real" values in the API tree (i.e., actual version numbers and not `null`/`true`/`false` values). It would be nice to see this become a real value before merging, or in a follow-up PR.

See https://github.com/mdn/browser-compat-data/issues/6369 for more about the goal.

Reply:
Updated it to `false` (checked implementation).